README.md

```diff
@@ -0,0 +1,45 @@
+Hm-admin
+--------
+
+[![Build Status](https://travis-ci.org/coopTilleuls/hm-admin.svg?branch=master)](https://travis-ci.org/coopTilleuls/hm-admin)
+[![Coverage Status](https://coveralls.io/repos/github/coopTilleuls/hm-admin/badge.svg?branch=master)](https://coveralls.io/github/coopTilleuls/hm-admin?branch=master)
+[![Issue Count](https://codeclimate.com/github/coopTilleuls/hm-admin/badges/issue_count.svg)](https://codeclimate.com/github/coopTilleuls/hm-admin)
+
+
+Hack with docker
```

Comment:
You should move this section in a `CONTRIBUTING.md` file. `README.md` should only present the project and how to install it.